cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -509,7 +514,7 @@ def test_service_ansible_playbook_pass_extra_vars(
     pre = stdout.text
     json_str = pre.split("--------------------------------")
     result_dict = json.loads(json_str[5].replace('", "', "").replace('\\"', '"').replace(
-        '\\, "', '",').split('" ] } PLAY')[0])
+        '\\, "', '",').replace("\\", "").split('" ] } PLAY')[0])
```

Comment:
**Suggestion:** Please add a comment about what are you trying to do here.  There are multiple replace calls, so it would be nice if you put what the string would look before and after you process it.

Reply:
I am still working on it. I will try to explain here as I am withdrawing string from Ansible Playbook logs.

Reply:
Added Comment there.